### PR TITLE
LPS-75159 Remove unused methods

### PR DESCRIPTION
**Pull request by pedroqueiroz94**

### CODE REVIEW

**Comment by pedroqueiroz94** [comment]
ci:test

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by pedroqueiroz94** [comment]
ci:retest

File: dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java [hunk 1]
```diff
@@ -532,6 +539,7 @@ protected boolean isShowSubmitButton() {
 	private final DDMFormInstanceRecordVersionLocalService
 		_ddmFormInstanceRecordVersionLocalService;
 	private final DDMFormInstanceService _ddmFormInstanceService;
+	private final DDMFormValuesMerger _ddmFormValuesMerger;
 	private final DDMFormRenderer _formRenderer;
```
Comment (leoadb) [inline dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java]: @pedroqueiroz94 Please add the **ddm** prefix

Comment (leoadb) [inline dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java]: @pedroqueiroz94 Please add the ddm prefix

**Comment by pedroqueiroz94** [comment]
@leoadb, I fixed the names. :)

**Pull request merged**